site/_posts/2020-08-26-announcing-fusionauth-1-19.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: blog-post
+title: Announcing FusionAuth 1.19
+description: The FusionAuth 1.19 release offers stateless, multi-tenant Single-Sign-on (SSO), more flexible configuration, and more.
+author: Dan Moore
+image: blogs/release-1-19/fusionauth-release-1-19.png
+category: blog
+excerpt_separator: "<!--more-->"
+---
+
+We're excited to announce the release of version 1.19 of FusionAuth, which shipped at the end of August. This version delivers new features and resolves issues for users running any older version.
+
+<!--more-->
+
+## Highlights
+
+In addition to bug fixes and user interface improvements, there are a few highlights we'd like to, well, highlight.
+
+### Revamped configuration options
+
+Configuration options are now consistent no matter how you set them. Every option can be specified in one of three ways:
```

Comment:
Let's say they are ordered here as well. It goes ENV then `-D` then `fusionauth.properties`.

Reply:
Great, thanks!